kolibri/plugins/coach/assets/src/views/elapsed-time/index.vue

```diff
@@ -0,0 +1,99 @@
+<template>
+
+  <span>{{ $tr($trUnit, {n: display}) }}</span>
+
+</template>
+
+
+<script>
+
+  const parse = require('date-fns/parse');
+  const differenceInSeconds = require('date-fns/difference_in_seconds');
+
+  const MINUTES_IN_DAY = 1440;
```

Comment:
Sort of like a local "constants" section. Is this a pattern you think I should start employing?